caldp/process.py

```diff
@@ -334,11 +343,47 @@ def find_input_files(self):
             some of which will be selected for calibration processing.
         """
         self.divider("Finding data files with glob *.fits for:", self.ipppssoot)
-        base_path = self.input_uri.split(":")[-1]
+        # find the base path to the files
+        test_path = self.input_uri.split(":")[-1]
+        if os.path.isdir(test_path):
```

Comment:
I actually think these two clauses are redundant, but I left them for now in case there's something I'm missing about os.path.isdir